openbb_platform/extensions/quantitative/openbb_quantitative/quantitative_router.py

```diff
@@ -137,7 +137,7 @@ def omega_ratio(
 
     epsilon = 1e-6  # to avoid division by zero
 
-    def get_omega_ratio(df_target: pd.Series, threshold: float) -> float:
+    async def get_omega_ratio(df_target: pd.Series, threshold: float) -> float:
```

Comment:
Does this one need to be also an `async`?

Reply:
yeah good suggestion, updated!